client/state/jetpack-settings/modules/test/actions.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	activateModule
+} from '../actions';
+
+import {
+	JETPACK_MODULE_ACTIVATE_SUCCESS
+} from 'state/action-types';
+import { useSandbox } from 'test/helpers/use-sinon';
+import wp from 'lib/wp';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+	} );
+
+	describe( '#activateJetpackModule', () => {
+		let sandbox;
+
+		useSandbox( newSandbox => sandbox = newSandbox );
+
+		it( 'should dispatch complete success when API activates a module', () => {
+			sandbox.stub( wp, 'undocumented', () => ( {
```

Comment:
Once we switch the `activateJetpackModule` to return a `Promise`, lets also use `nock` to emulate the http requests.